DOCUMENTATION.md

```diff
@@ -421,11 +467,9 @@ Below are all functions that don't have anything to do with apps.
   ~/pi-apps/api get_device_info
   ```
   - This function is used in the `format_log_file` function.
-- `functions_to_files` - Takes every function in the `api` and turns them into their own miniature bash scripts.
-  - This exists purely for developer-convenience. It allows you to handle functions as if they were files.
-  - It creates a folder (`~/pi-apps/function-files`) and then places files in it.
-- `files_to_functions` - Takes every file in the `function-files` folder and re-combines them.
-  - The resulting output is printed to the terminal.
+- `get_codename` - Simple function to return the codename of the operating system.
+  - On Raspberry Pi OS Bullseye, this function returns: `bullseye`
+  - This is useful for scripts that need to do different things depending on the exact system they are running on. The MultiMC5 app is a great example of this.
```

Comment:
fyi, yes the MultiMC app does different things depending on the exact system, but it does not use the get_codename function. That was added by someone else later and doesn't have enough info to do what the script needs done.

Reply:
I mentioned MultiMC5 because it seems to do something completely different on every OS! 🤣
However, if you would prefer to use a different app, you are free to suggest one.

Reply:
Its fine to use.
Though the path is pretty simple:
- Determine if Debian or Ubuntu based
- if Debian and (bullseye|buster|stretch|jessie), add temurin java deb repos and install packages. If not (bullseye|buster|stretch|jessie), error and exit.
- if Ubuntu and (bionic|focal|groovy), add openjdk ppa and install packages. If above groovy, skip ppa add and just install packages (ppa is not needed). If below bionic, error and exit.